apps/docs/src/app/core/component-docs/list/examples/list-action-example/list-action-example.component.html

```diff
@@ -5,7 +5,7 @@
     <li fd-list-item *ngFor="let i of items">
         <span fd-list-title> List Item {{ i }} </span>
     </li>
-    <li fd-list-item [action]="true" [growing]="true">
+    <li fd-list-item [action]="true" [growing]="true" tabindex="-1" aria-hidden="true">
```

Comment:
Please explain in more details why you did this, esp the part where you add tabindex to the `li` element. The content of the li element is a button that takes the whole width and height and is by default focussable and what concerns the user is the content inside the button, so I really struggle to understand why we need to focus the `li` element.

Reply:
This list-item element has a button inside, both list-item and inner button can receive focus. Because of this you have to actually hit TAB 2 times to go through this element. So I've added these attributes in order to omit the focus on list-item. 

The problem is currently it's supported to add multiple elements inside the tab, while it's possible that none of the descendants will be focusable. So not sure what would be the better solution

Reply:
> 

In which examples does the list-item element have a button inside? Not in this one, and only one tab press is needed to move through the items on the current implementation https://fundamental-ngx.netlify.app/#/core/list#action

Reply:
@mikerodonnell89 you can see it in this `Action list` example. 


https://user-images.githubusercontent.com/33101123/145172327-d86c297b-463e-49ec-b20a-96c6749b1c5f.mp4



However, I removed `aria-hidden` now, since this change is so questionable. It was added in order to not include list item with button in the total number of items.

Reply:
There is not much sence to have li element focusable at all, maybe we should remove `tabindex=0` from there?

Reply:
@platon-rov don't think so: we're not limiting the content of `fd-list-item` to be something specific, so it can be something that is not focusable. Because of this the behavior of the list might be not that predictable

Reply:
@dmitry-stepanenko okay, so let' imagine we have only text in `fd-list-item` then for what reason we need `tabindex=0` on that item?

Reply:
I saw the pr that makes only first item focusable and next are accessible using arrows but cannot find it right now.

Reply:
Ahaha, that's current pr, that's the reason i didn't find it :D